src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static Color getRandomColor() {
+        return Color.values()[new Random().nextInt(Color.values().length)];
```

Comment:
the same comments that I left in the previous HW. 
Don’t create a static method. Make Random class-level variable